bun/bun.go

```diff
@@ -74,6 +75,7 @@ func MakeBunHandler(pathPrefix, commit, date string) mux.Mux {
 	m.HandleFunc("/html", handleSampleHtml)
 	m.HandleFunc("/json", handleSampleJson)
 	m.HandleFunc("/robots.txt", handleSampleRobotsTxt)
+	m.HandleFunc("/env", handleEnv)
```

Comment:
As discussed on the linked issue, let's move this into `/info` and remove this line.